### PR TITLE
Add subgame Parachain/Network to polkadotjs 

### PR DESCRIPTION
**Pull request by momo200e**
* Adds subgame to Kusama parachain
* Adds subgame_staging to testnet
* Adds subgame to Network

### CODE REVIEW

File: packages/apps-config/src/api/spec/subgame.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        },
+        BetInfo: {
+          user: 'AccountId',
+          game_id: 'u32',
+          amount: 'u128',
+          game_mode: 'u8' // 1 = odd , 2 = even
+        },
+        Template: {
+          template_id: 'u32',
+          template_name: 'TemplateName'
```
Comment (jacogr) [inline packages/apps-config/src/api/spec/subgame.ts:40]: The `TemplateName` type is not defined in these definitions.

**Comment by momo200e** [comment]
@jacogr  Thank you very much for your reminder, I have updated it

File: packages/apps-config/src/api/spec/subgame.ts (hunk 1 above)
Comment (momo200e) [reply packages/apps-config/src/api/spec/subgame.ts]: Done

**Comment by momo200e** [comment]
@jacogr  Sorry, I am not sure which issues have not been resolved. Can you confirm for me? Thanks

**Comment by antonkhvorov** [comment]
Hey @momo200e !
I'm from fearlesswallet.io team
We are supporting native crowdloans as well, so SubGame crowdloan is available for our users already.
To saturate your crowdloan with info, we need to understand your reward rate (what users will receive for each 1 KSM locked).
Do you have any link describing your crowdloan campaign?

**Review by jacogr — APPROVED** [review]
Thank you.

I would suggest just running `yarn lint` locally with any changes, it is much easier with auto-fix instead of waiting for the CI cycle.

**Pull request merged**